docs/content/hdfs.md

```diff
@@ -90,6 +90,26 @@ Sync the remote `directory` to `/home/local/directory`, deleting any excess file
 
     rclone sync -i remote:directory /home/local/directory
 
+### Own HDFS instance ###
+
+You may start with (manual)[https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html] or use docker image from tests:
+```
+cd fstest/testserver/images/test-hdfs
```

Comment:
Normally users don't have rclone source installed.
Let's either prepend
```
git clone https://github.com/rclone/rclone.git
cd rclone
```
or let's ask @ncw to push `rclone/test-hdfs` up https://hub.docker.com/u/rclone and remove build step from docs.

Reply:
updated.